snakemake/script/__init__.py

```
@@ -30,7 +30,7 @@
 from snakemake.common import MIN_PY_VERSION, ON_WINDOWS, get_snakemake_searchpaths
 from snakemake.deployment import singularity
 from snakemake.exceptions import WorkflowError
-from snakemake.logging import logger
+from snakemake.logging import logger_manager, logger
```

Ruff (0.8.2)

`snakemake.logging.logger_manager` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)